Altis_Life.Altis/core/actions/fn_gather.sqf

```diff
@@ -6,51 +6,59 @@
     Description:
     Main functionality for gathering.
 */
-private ["_maxGather","_resource","_amount","_maxGather","_requiredItem"];
+
+scopeName "main";
```

Comment:
scopeName hasn't been used anywhere else so far, should we move toward using scopeName in all scripts - or keep it consistent and continue without